src/components/events/EventsSection.jsx

```diff
@@ -18,23 +19,29 @@ class EventsSection extends React.Component {
   }
 
   componentDidMount() {
-    db.collection('events').onSnapshot((querySnapshot) => {
-      const events = [];
-      querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, eventDate } = doc.data();
+    const events=[];
```

Comment:
can we revert the spacing on this line

Reply:
line 22 was moved to a different section of the code... this doesn't seem to be reflected in the changes shown above